ee/dataflatten/flatten.go

```diff
@@ -141,7 +142,7 @@ func Flatten(data interface{}, opts ...FlattenOpts) ([]Row, error) {
 	}
 
 	if !fl.debugLogging {
```

Comment:
This boolean was kinda weird.

IIRC I added it (and the associated `WithDebugLogging`) because I didn't want the flatten code to log most of the time. I want logs when I'm debugging flattening, and otherwise they're kinda noisy and in the way. Not sure if there's a better pattern

Reply:
If I were doing it from scratch, I
Might write the logs at a level below debug. Maybe. 

I think you could remove the conditional, and leave it as effectively unused. We can either rip out logs or relevel them in a follow up.

Reply:
Yup, that handles it!